PhpAmqpLib/Wire/IO/StreamIO.php

```diff
@@ -59,6 +62,7 @@ class StreamIO extends AbstractIO
      * @param null $context
      * @param bool $keepalive
      * @param int $heartbeat
+     * @param int $amqpHeaderAsInt
```

Comment:
I think it would be better for optional parameters to not be on the constructor and use a setter instead.

Also would it make sense to name it something like `$maxFrameSize`?